rosbag2_storage_mcap/package.xml

```diff
@@ -8,13 +8,16 @@
   <license>Apache-2.0</license>
 
   <buildtool_depend>ament_cmake</buildtool_depend>
-  <buildtool_depend>python3-pip</buildtool_depend>
 
+  <buildtool_export_depend>lz4_cmake_module</buildtool_export_depend>
+  <build_export_depend>lz4</build_export_depend>
+
+  <depend>ament_index_cpp</depend>
+  <depend>mcap_vendor</depend>
   <depend>pluginlib</depend>
   <depend>rcutils</depend>
   <depend>ros_environment</depend>
   <depend>rosbag2_storage</depend>
-  <depend>ament_index_cpp</depend>
```

Comment:
I don't see the `fmt` or `zstd_vendor` dependencies in the package.xml.

I think that they need to be at least `build_depend`s since I do not see either header being exported in CMake, but if the mcap public interface includes interfaces from zstd or fmt then they should be global `depend` or `build_export_depend`s.

It may be that libfmt-dev doesn't yet have a rosdep key. That's something I can help with.

Reply:
FYI we removed the fmt dependency from mcap, since it was not too much effort: https://github.com/foxglove/mcap/pull/359

Reply:
addressed in https://github.com/ros-tooling/rosbag2_storage_mcap/pull/26/commits/dd211a6f1c58e7666bcdc1497a8a3131b494b67d and https://github.com/ros-tooling/rosbag2_storage_mcap/pull/26/commits/a02fddc266bf1f20b354368a9739c9317d6be274